examples/sync-merkleroots.ts

```diff
@@ -0,0 +1,92 @@
+import {
+  ErrorResponse,
+  MerkleRoot,
+  MerkleRootsRepository,
+  SpvWalletClient,
+  SpvWalletError,
+} from '../dist/typescript-npm-package.cjs.js';
+import { exampleXPub } from './example-keys.js';
+import { errMessage } from './utils.js';
+
+const server = 'http://localhost:3003/api/v1';
+
+// perhaps change exmapleXPub to actual existing in db
+if (!exampleXPub) {
+  console.log(errMessage('xPub'));
+  process.exit(1);
+}
+
+const client = new SpvWalletClient(server, {
+  xPub: exampleXPub,
```

Comment:
please use xpriv

Reply:
In spv-wallet we are checking for the xpub or access-key, why do you want to use xpriv here?

Reply:
Authorizing via xPriv also sends xpub via network but also the content is signed. 
We should "almost always" use the xPriv method.

Reply:
changed